std/algorithm/sorting.d

```diff
@@ -844,7 +844,7 @@ private template validPredicates(E, less...)
 }
 
 /**
-$(D void multiSort(Range)(Range r)
+$(D auto multiSort(Range)(Range r)
```

Comment:
is there a way how we could write `SortedRange` here?
The `less` fun is defined inline.